src/temperature_component.cpp

```diff
@@ -10,6 +10,10 @@
  *
  *  Created by Ben Vega-Westhoff on 11/1/16.
  *
+ * Meinshausen, M., Raper, S. C. B., and Wigley, T. M. L.: Emulating coupled atmosphere-ocean
```

Comment:
I added it because in line 239 we use an equation to get the forcing per CO2 doubling from the radiative forcing from forcing efficiency for CO2. But good point we should also add the citations for DOECLIM.